config/rbac/role.yaml

```diff
@@ -21,6 +21,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
```

Comment:
```suggestion
  - get
  - list
  - watch
```
We all three due to informer cache.
It still gave error with just `get` verb

Reply:
ah, ok, will add them